lib/accessory.js

```diff
@@ -54,15 +64,19 @@ class Accessory {
   }
 
   setOutlet (uuid, powered, done) {
-    this.remoteOutlet(uuid).update({desired_state: {powered}}, () => {
+	  var that = this;
+    this.remoteOutlet(uuid).update({desired_state: {powered}}, () => {	
+		that.outlets.map(function(outlet){
```

Comment:
No worries, I linked out template litterals in one of my comments above, that describes is